docs/image-sizes.md

```diff
@@ -0,0 +1,40 @@
+# Images Sizes
+
+You can define additional image thumbnail sizes with `add_image_size`.
+
+The custom thumbnail sizes should be declared in the callback function to the `after_setup_theme` action.
```

Comment:
You should document the function `add_image_size()` and the parameters it takes before this. Right now the example below doesn't call special attention to the `add_image_size()` so it's not as scannable as it could be.

Try and think that people reading this might have no prior WordPress or Altis knowledge.